src/screens/Game/SectionSummary/SectionSummary.tsx

```diff
@@ -1,21 +1,222 @@
-import { View } from "react-native";
+import React from "react";
+import { View, Text } from "react-native";
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import Text from "../../../components/Text";
-import Button from "../../../components/Button";
-import { RootStackParamList } from "../../../types";
+import FontAwesome5 from "react-native-vector-icons/FontAwesome5";
+import { useSelector } from "react-redux";
+import { GameDetails, RootStackParamList } from "../../../types";
+import { RootState } from "../../../redux/rootReducer";
 
-type Props = NativeStackScreenProps<RootStackParamList, "ExercisesCompleted">;
+type Props = NativeStackScreenProps<RootStackParamList, "SectionSummary">;
 
-export default function SectionSummary({ navigation }: Props) {
+interface StatComponentProps {
+  title: string;
+  stat: string;
+  statColor: string;
+}
+
+function StatComponent({ title, stat, statColor }: StatComponentProps) {
```

Comment:
We don't need this StatComponent, as we already have OnelineComponent and TwoLineComponent for you to use. Could you fix this to use the component boxes